backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3333,4 +3333,102 @@ databaseChangeLog:
                   constraints:
                     nullable: true
                     foreignKeyName: fk__facility__default_specimen_type
-                    references: specimen_type(internal_id)
\ No newline at end of file
+                    references: specimen_type(internal_id)
+  - changeSet:
+      id: add-disease-table
+      author: emma@skylight.digital
+      comment: Adds a non-audited table to track which diseases SimpleReport supports
+      changes:
+        - tagDatabase:
+            tag: add-supported-disease-table
+        - createTable:
+            tableName: supported_diseases
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: text
+                  remarks: Tracks the human-readable name for the disease
+                  constraints:
+                    nullable: false
+              - column:
+                  name: loinc
+                  type: text
+                  remarks: Tracks the LOINC code for this disease
+                  constraints:
+                    nullable: false
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.supported_diseases TO ${noPhiUsername};
+      rollback:
+          sql: |
+            DROP TABLE ${database.defaultSchemaName}.supported_diseases;
+  - changeSet:
+      id: populate-supported-diseases-table
```

Comment:
same here `diseases` vs `disease` in the tag.

Reply:
Updated to `disease` everywhere